bin/query-db-and-email

```diff
@@ -167,31 +167,36 @@ function writeCsvsForSites (pagesBySite) {
 }
 
 function csvStringForPages (pages) {
-  let pageIndex = 1;
-  const rows = pages.map(page => {
-    const version = page.latest;
-    const metadata = version.source_metadata;
-    return [
-      pageIndex++,
-      version.uuid,
-      // TODO: format
-      timeString,
-      page.agency,
-      page.site,
-      page.title,
-      page.url,
-      `https://versionista.com/${metadata.site_id}/${metadata.page_id}/`,
-      metadata.diff_with_previous_url,
-      metadata.diff_with_first_url,
-      // TODO: format
-      formatCsv.formatDate(new Date(version.capture_time)),
-      '----',
-      metadata.diff_length,
-      metadata.diff_hash,
-      metadata.diff_text_length,
-      metadata.diff_text_hash
-    ];
-  });
+  const rows = pages
+    .map(page => {
+      const version = page.latest;
+      const metadata = version.source_metadata;
+      return [
+        '',
+        version.uuid,
+        // TODO: format
+        timeString,
+        page.agency,
+        page.site,
+        page.title,
+        page.url,
+        `https://versionista.com/${metadata.site_id}/${metadata.page_id}/`,
+        metadata.diff_with_previous_url,
+        metadata.diff_with_first_url,
+        // TODO: format
+        formatCsv.formatDate(new Date(version.capture_time)),
+        '----',
+        metadata.diff_length,
+        metadata.diff_hash,
+        metadata.diff_text_length,
+        metadata.diff_text_hash
+      ];
+    })
+    .sort(formatCsv.sortCsv)
+    .map((value, index) => { //sets Index column
+      value[0] = index + 1;
+      return value;
+    });
```

Comment:
Having this repeated block in multiple places (where it would never be changed independently) implies that it isn’t as well-factored as it could be. You could wrap it up in a clearly named function (e.g. `addIndexesToRows`), but I think you could also argue that this is a fundamental part of sorting, so the whole `.sort(comparator).map(functionThatSetsIndexes)` operation should be a single function call.